src/Blaster.WebApi/Features/Shared/components/Shared.js

```diff
@@ -6,6 +8,17 @@ import ChannelIconComponent from "./ChannelIconComponent";
 import ChannelInputComponent from "./ChannelInputComponent";
 import BannerComponent from "./BannerComponent";
 
+Vue.use(UserManagementPlugin);
+
+new Vue({
+    el: ".navbar",
+    computed: {
+        showIEBanner: function () {
```

Comment:
*showIEBanner* doesn't seem to be used for anything here. Would make sense to put it here though, instead of adding it page by page. Funnily enough, this PR breaks IE11 support.

Reply:
The problem is from the vue-msal component. The showIEBanner functionality is just fine. :)